src/parser/classes/VideoSecondaryInfo.ts

```diff
@@ -23,7 +24,8 @@ class VideoSecondaryInfo extends YTNode {
     this.owner = Parser.parseItem<VideoOwner>(data.owner);
     this.description = new Text(data.description);
     this.subscribe_button = Parser.parseItem<SubscribeButton | Button>(data.subscribeButton, [ SubscribeButton, Button ]);
-    this.metadata = Parser.parseItem<MetadataRowContainer>(data.metadataRowContainer, MetadataRowContainer);
+    // MetadataRowContainer is redundant, we want to target 'RichMetadataRow'
+    this.metadata = Parser.parseItem<MetadataRowContainer>(data.metadataRowContainer, MetadataRowContainer)?.rows?.[0] ?? null;
```

Comment:
Could replace `rows?.[0] ?? null` with `rows.firsfOfType(RichMetadataRow)`. Also, couldn't there be more `RichMetadataRow` items other than the first one?